py/ai/h2o/sparkling/ml/params/H2OAutoMLParams.py

```diff
@@ -27,26 +26,26 @@ class H2OAutoMLParams(H2OCommonSupervisedParams):
+    ignoredCols = Param(Params._dummy(), "ignoredCols", "Ignored column names", H2OTypeConverters.toListString())
     includeAlgos = Param(Params._dummy(), "includeAlgos", "Algorithms to include when using automl",
                          H2OTypeConverters.toEnumListString("ai.h2o.automl.Algo", nullEnabled=True))
     excludeAlgos = Param(Params._dummy(), "excludeAlgos", "Algorithms to exclude when using automl",
                          H2OTypeConverters.toEnumListString("ai.h2o.automl.Algo", nullEnabled=True))
     projectName = Param(Params._dummy(), "projectName", "identifier for models that should be grouped together in the leaderboard" +
-                        " (e.g., airlines and iris)", TypeConverters.toString)
-    maxRuntimeSecs = Param(Params._dummy(), "maxRuntimeSecs", "Maximum time in seconds for automl to be running", TypeConverters.toFloat)
-    stoppingRounds = Param(Params._dummy(), "stoppingRounds", "Stopping rounds", TypeConverters.toInt)
-    stoppingTolerance = Param(Params._dummy(), "stoppingTolerance", "Stopping tolerance", TypeConverters.toFloat)
+                        " (e.g., airlines and iris)", H2OTypeConverters.toNullableString())
+    maxRuntimeSecs = Param(Params._dummy(), "maxRuntimeSecs", "Maximum time in seconds for automl to be running", H2OTypeConverters.toFloat())
+    stoppingRounds = Param(Params._dummy(), "stoppingRounds", "Stopping rounds", H2OTypeConverters.toInt())
+    stoppingTolerance = Param(Params._dummy(), "stoppingTolerance", "Stopping tolerance", H2OTypeConverters.toFloat())
     stoppingMetric = Param(Params._dummy(), "stoppingMetric", "Stopping metric",
                            H2OTypeConverters.toEnumString("hex.ScoreKeeper$StoppingMetric"))
     sortMetric = Param(Params._dummy(), "sortMetric", "Sort metric for the AutoML leaderboard",
                        H2OTypeConverters.toEnumString("ai.h2o.sparkling.ml.algos.H2OAutoMLSortMetric"))
-    balanceClasses = Param(Params._dummy(), "balanceClasses", "Balance classes", TypeConverters.toBoolean)
-    classSamplingFactors = Param(Params._dummy(), "classSamplingFactors", "Class sampling factors", TypeConverters.toListFloat)
-    maxAfterBalanceSize = Param(Params._dummy(), "maxAfterBalanceSize", "Max after balance size", TypeConverters.toFloat)
-    keepCrossValidationPredictions = Param(Params._dummy(), "keepCrossValidationPredictions", "Keep cross validation predictions", TypeConverters.toBoolean)
-    keepCrossValidationModels = Param(Params._dummy(), "keepCrossValidationModels", "Keep cross validation models", TypeConverters.toBoolean)
-    maxModels = Param(Params._dummy(), "maxModels", "Max models to train in AutoML", TypeConverters.toInt)
+    balanceClasses = Param(Params._dummy(), "balanceClasses", "Balance classes", H2OTypeConverters.toBoolean())
+    classSamplingFactors = Param(Params._dummy(), "classSamplingFactors", "Class sampling factors", H2OTypeConverters.toNullableListFloat())
```

Comment:
Please can help me understand why `classSamplingFactors` can contains `null`s?

Reply:
Class sampling factors can contain `None` as H2O's code distinguished between null and empty array. The elements itself can't be null